dali/test/python/test_dali_cpu_only.py

```diff
@@ -64,7 +64,10 @@ def get_data():
     pipe = Pipeline(batch_size=batch_size, num_threads=3, device_id=None)
     outs = fn.external_source(source=get_data)
     pipe.set_outputs(outs.gpu())
-    assert_raises(RuntimeError, pipe.build)
+    assert_raises(
+        RuntimeError,
+        pipe.build,
+        glob="Cannot make a gpu output for __ExternalSource_0 operator, device_id should not be equal CPU_ONLY_DEVICE_ID.")
```

Comment:
This message doesn't sport the most beautiful English. Indeed, I don't even know what it means and what could be understood by the user.
There are other problems with this message - for once, the name used in the message is not the name of the operator/producer, but the name of the output (a DataNode).
```suggestion
        glob='Cannot move the data node  \"__ExternalSource_0\" to the GPU in a CPU-only pipeline.'
The `device_id` parameter is set tot `CPU_ONLY_DEVICE_ID`. Set `device_id` to a valid GPU identifier to enable GPU features in the pipeline. ")
```

Reply:
Done